mailer.js

```diff
@@ -61,8 +67,8 @@ module.exports = function (log) {
     this.mailer.close()
   }
 
-  Mailer.prototype._supportLinkAttributes = function () {
-    return linkAttributes(this.supportUrl)
+  Mailer.prototype._supportLinkAttributes = function (template) {
+    return linkAttributes(this._generateUTMLink(this.supportUrl, {}, template, 'support'))
```

Comment:
Ahha, I was confused, your use makes perfect sense. I want the utm parameter on the [passwordChangeLink](https://github.com/mozilla/fxa-auth-mailer/pull/190/files/22e7a0840540ca104090e636f3529c73c1ff8766#diff-3427c5e3d78d8d6550178ad46b9e0680R234) and [passwordResetLink](https://github.com/mozilla/fxa-auth-mailer/pull/190/files/22e7a0840540ca104090e636f3529c73c1ff8766#diff-3427c5e3d78d8d6550178ad46b9e0680R290).